src/index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-thu calendar--month-30">
```

Comment:
```suggestion
    <div class="calendar calendar--start--thu calendar--month--30">
```

Reply:
> 

Sorry, my bad, I thought --start-day was the modifier. It turns out that there will be 2 of them now? --start and --day